tools/src/wyvern/target/corewyvernIL/expression/Bind.java

```diff
@@ -66,4 +67,16 @@ public Value interpret(EvalContext ctx) {
 		Value v = toReplace.interpret(ctx);
 		return inExpr.interpret(ctx.extend(varName, v));
 	}
+
+	@Override
+	public Set<String> getFreeVariables() {
```

Comment:
Technically bind has exactly the free variables that are in the "toReplace" part.  The inExpr can use varName, and nothing else.